src/io/pithos/xml.clj

```diff
@@ -400,6 +400,10 @@ Will produce an XML AST equivalent to:
         [:BucketName (:bucket payload)]
         [:RequestId reqid]
         [:HostId reqid]]
+       :no-such-cors
+       [:Error
+        [:Code "NoSuchCORSConfiguration"]
+        [:Message "The CORS configuration does not exist"]]
```

Comment:
Can you add `BucketName` to the xml to match AWS S3 responses?

Other inconsistencies with S3 API:

* In NoSuchBucketPolicy the `Bucket` key should be `BucketName`.
* get bucket lifecycle should return a 404 when lifecycle configuration is empty (`<Error><Code>NoSuchLifecycleConfiguration</Code><Message>The lifecycle configuration does not exist</Message><BucketName>yo</BucketName>`)